scripts/submitBatch.js

```diff
@@ -0,0 +1,44 @@
+const { ethers } = require("ethers");
+const contractAddresses = require("../contractAddresses.json");
+
+const RollupContract = require("../build/contracts/Rollup.json");
+
+const argv = require("minimist")(process.argv.slice(2), {
+    string: ["batches"]
+});
+
+/*
+    $ node ./scripts/submitBatch.js --batches batches.json
```

Comment:
Need the format of `batches.json`

Reply:
Getting from https://gist.githubusercontent.com/vaibhavchellani/3c1f19925cf2d3a26edf218ff7cff33d/raw/7f4234f7daa6e99043f095df477d93c76708c504/commitments.json